Offices/Offices.Application/Extensions/RepositoriesExtension.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Offices.Data.Repositories;
+
+namespace Offices.Application.Extensions
+{
+    public static class RepositoriesExtension
```

Comment:
This should be in the Data layer like everything related to the interaction with the db

Reply:
Fixed